utils/jenkins/scripts/common.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2019-2020, Intel Corporation
+
+# common.sh - contains bash functions used in all jenkins pipelines.
+
+set -o pipefail
+
+scriptdir=$(readlink -f $(dirname ${BASH_SOURCE[0]}))
+
+function system_info {
+	echo "********** system-info **********"
+	cat /etc/os-release | grep -oP "PRETTY_NAME=\K.*"
+	uname -r
+	echo "libndctl: $(pkg-config --modversion libndctl || echo 'libndctl not found')"
+	echo "libfabric: $(pkg-config --modversion libfabric || echo 'libfabric not found')"
+	echo "libpmem: $(pkg-config --modversion libpmem || echo 'libpmem not found')"
+	echo "libpmemobj: $(pkg-config --modversion libpmemobj || echo 'libpmemobj not found')"
+	echo "libpmemobj++,: $(pkg-config --modversion libpmemob++j || echo 'libpmemobj not found')"
```

Comment:
there are also two issues next to "libpmemobj++":
1. there's weird "," at the beginning, and
2. there's definitely unnecessary "j" in pkg-config command param